backend/src/packages/auth/auth.service.ts

```diff
@@ -52,6 +53,13 @@ class AuthService {
       user,
     };
   }
+
+  public async getCurrent(token: string): Promise<UserAuthResponse> {
+    const { userId } = this.tokenService.decode<{ userId: number }>(token);
+    const user = await this.userService.find(userId);
+
+    return user as UserAuthResponse;
```

Comment:
```suggestion
    return user;
```
handle if `userService.find` returns `null`